tests/test_IDL_Type.py

```diff
@@ -5,11 +5,51 @@
 import lsst.ts.xml as ts_xml
 
 
-def check_for_issues(csc, topic):
-    jira = ""
+def check_for_issues(test, csc, topic):
+    if test == "exists" and csc == "ATPtg" and topic == "Telemetry":
+        jira = "DM-27120"
+    elif test == "exists" and csc == "MTPtg" and topic == "Telemetry":
+        jira = "DM-27121"
+    else:
+        jira = ""
```

Comment:
Please consider separate check_for_issues functions for exists and string (e.g. `check_for_type_issues` and `check_for_string_units_issues`). I think it would clarify the logic.

In fact I gently suggest moving the "strings should not have units" test to its own separate file. It's a very different test than testing that IDL_Type exists and is borderline between this file and test_Units.py.

Reply:
I'm going to leave this on as-is.  I prefer fewer functions, and I think this is the way it's done in other test modules.  I appreciate that the 'exists' test is somewhat fluid between IDL_Type and Units, but if/when the schema tests work correctly, the exists test could be entirely removed.